.travis.yml

```diff
@@ -24,6 +24,7 @@ install:
   - pip install -q -r test_requirements.txt
   - pip install -q pylint pycodestyle
   - pip install -q torch==1.3.1+cpu torchvision==0.4.2+cpu -f https://download.pytorch.org/whl/torch_stable.html
+  - pip install cma==2.7.0
```

Comment:
Please move this dependency to test_requirements.txt.

Reply:
Moved.

Reply:
Moved the dependency.